test/integration/src/test/java/io/pravega/test/integration/CheckpointTest.java

```diff
@@ -77,7 +83,7 @@ public static void teardown() {
 
     @Test(timeout = 20000)
     public void testCheckpointAndRestore() throws ReinitializationRequiredException, InterruptedException,
-                                           ExecutionException, TimeoutException {
```

Comment:
changes not required here.

Reply:
There is no modification to this testcase only overwriting created this difference.